gnovm/pkg/gnolang/preprocess.go

```diff
@@ -4242,6 +4740,12 @@ func SetNodeLocations(pkgPath string, fileName string, n Node) {
 	})
 }
 
+// XXX check node lines, uniqueness of locations,
+// and also check location pkgpath and filename.
```

Comment:
add comment // Even after this is implemented, locations should not be used for logic.

Reply:
done